Stuart Hughes stuarth@freescale.net 22mar05

Allow to be build as non-root (for rpm builds)

diff --exclude CVS -uNr tcp_wrappers_7.6/Makefile tcp_wrappers_7.6.modified/Makefile
--- tcp_wrappers_7.6/Makefile	2005-03-22 05:50:22.000000000 -0600
+++ tcp_wrappers_7.6.modified/Makefile	2005-03-22 05:50:05.000000000 -0600
@@ -774,28 +774,28 @@
 install: install-lib install-bin install-dev
 
 install-lib:
-	install -o root -g root -m 0755 $(SHLIB) ${DESTDIR}/usr/lib/
+	install -m 0755 $(SHLIB) ${DESTDIR}/usr/lib/
 	ln -sf $(notdir $(SHLIB)) ${DESTDIR}/usr/lib/$(notdir $(SHLIBSOMAJ))
 	ln -sf $(notdir $(SHLIBSOMAJ)) ${DESTDIR}/usr/lib/$(notdir $(SHLIBSO))
 
 install-bin:
-	install -o root -g root -m 0755 tcpd ${DESTDIR}/usr/sbin/
-	install -o root -g root -m 0755 tcpdchk ${DESTDIR}/usr/sbin/
-	install -o root -g root -m 0755 tcpdmatch ${DESTDIR}/usr/sbin/
-	install -o root -g root -m 0755 try-from ${DESTDIR}/usr/sbin/
-	install -o root -g root -m 0755 safe_finger ${DESTDIR}/usr/sbin/
-	install -o root -g root -m 0644 tcpd.8 ${DESTDIR}/usr/share/man/man8/
-	install -o root -g root -m 0644 tcpdchk.8 ${DESTDIR}/usr/share/man/man8/
-	install -o root -g root -m 0644 try-from.8 ${DESTDIR}/usr/share/man/man8/
-	install -o root -g root -m 0644 tcpdmatch.8 ${DESTDIR}/usr/share/man/man8/
-	install -o root -g root -m 0644 safe_finger.8 ${DESTDIR}/usr/share/man/man8/
-	install -o root -g root -m 0644 hosts_access.5 ${DESTDIR}/usr/share/man/man5/
-	install -o root -g root -m 0644 hosts_options.5 ${DESTDIR}/usr/share/man/man5/
+	install -m 0755 tcpd ${DESTDIR}/usr/sbin/
+	install -m 0755 tcpdchk ${DESTDIR}/usr/sbin/
+	install -m 0755 tcpdmatch ${DESTDIR}/usr/sbin/
+	install -m 0755 try-from ${DESTDIR}/usr/sbin/
+	install -m 0755 safe_finger ${DESTDIR}/usr/sbin/
+	install -m 0644 tcpd.8 ${DESTDIR}/usr/share/man/man8/
+	install -m 0644 tcpdchk.8 ${DESTDIR}/usr/share/man/man8/
+	install -m 0644 try-from.8 ${DESTDIR}/usr/share/man/man8/
+	install -m 0644 tcpdmatch.8 ${DESTDIR}/usr/share/man/man8/
+	install -m 0644 safe_finger.8 ${DESTDIR}/usr/share/man/man8/
+	install -m 0644 hosts_access.5 ${DESTDIR}/usr/share/man/man5/
+	install -m 0644 hosts_options.5 ${DESTDIR}/usr/share/man/man5/
 
 install-dev:
-	install -o root -g root -m 0644 hosts_access.3 ${DESTDIR}/usr/share/man/man3/
-	install -o root -g root -m 0644 tcpd.h ${DESTDIR}/usr/include/
-	install -o root -g root -m 0644 $(LIB) ${DESTDIR}/usr/lib/
+	install -m 0644 hosts_access.3 ${DESTDIR}/usr/share/man/man3/
+	install -m 0644 tcpd.h ${DESTDIR}/usr/include/
+	install -m 0644 $(LIB) ${DESTDIR}/usr/lib/
 	ln -sf hosts_access.3 ${DESTDIR}/usr/share/man/man3/hosts_ctl.3
 	ln -sf hosts_access.3 ${DESTDIR}/usr/share/man/man3/request_init.3
 	ln -sf hosts_access.3 ${DESTDIR}/usr/share/man/man3/request_set.3
